sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-apollo/pom.xml

```diff
@@ -48,5 +48,10 @@
             <version>${apollo.client.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jul-to-slf4j</artifactId>
+            <scope>compile</scope>
```

Comment:
`compile` is a default scope, we do not need add it if scope is `compile`

Reply:
Thanks, I have fixed it and remember.